.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -981,7 +978,7 @@ stages:
       artifact: linux-monitoring-home-$(artifactSuffix)
 
 - stage: build_macos
-  condition: and(succeeded(), eq(variables['isBenchmarksOnlyBuild'], 'False'))
+  condition: and(succeeded())
```

Comment:
You can find-replace all instances of this, as `condition: succeeded()` is the default 🙂 
```suggestion
```

Reply:
Hmm I don't have any of these locally 😕